airflow/executors/__init__.py

```diff
@@ -22,7 +22,12 @@
 
 try:
     from airflow.executors.celery_executor import CeleryExecutor
-except:
+except ImportError:
+    pass
+
+try:
```

Comment:
Shouldn't we handle this the same as the MesosExecutor (CeleryExecutor should be updated as well), ie. make it dependent one the configuration if we should load the module and not dependent whether it is available.